_data/merchants.yml

```diff
@@ -331,8 +327,6 @@
       url: https://drippinjunkies.co.uk/
     - name: Ecosystems retail - new & refurbished IT equipment
       url: https://store.ecosystems-group.co.uk
-    - name: FastTech
-      url: https://www.fasttech.com/
```

Comment:
Looks like they still accept Monero for payments (see the footer)

Reply:
It's not an option in the checkout page:

![fasttech](https://user-images.githubusercontent.com/77471692/105275839-262dd480-5b6e-11eb-9ac6-a13f0c473581.png)